src/IssueViz/Commands/IssueVizPackage.vsct

```diff
@@ -117,5 +119,9 @@
       <IDSymbol name="grpDaemonErrorList" value="0x100" />
     </GuidSymbol>
 
+    <GuidSymbol name="SonarLintCmdSet" value="{DB0701CC-1E44-41F7-97D6-29B160A70BCB}">
```

Comment:
I'm not a big fan of it but I don't have another solution right now. We might want to revisit it when we have more windows/commands under the root SonarLint menu. Right now I cannot set the priority of this, it only looks ok because the ConnectedMode and Help both have a priority, so this one is in the middle by default.